modules/machine-pools-hcp.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa_nodes/rosa-nodes-machinepools-about.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="machine-pools-hcp_{context}"]
+= Machine pools in {hcp-title} clusters
+
+In {hcp-title} clusters, the {hcp} spans three availability zones (AZ) in the installed cloud region. Each machine pool in a {hcp-title} cluster deploys in a single subnet within a single AZ. Each of these AZs can only have one machine pool. 
```

Comment:
Per the [ISG](https://www.ibm.com/docs/en/ibm-style?topic=grammar-adverbs-only) we should move "only" as close to the phrase it modifies if possible. If this sentence means that AZs are limited to one machine pool, I'd change it to this:
```suggestion
In {hcp-title} clusters, the {hcp} spans three availability zones (AZ) in the installed cloud region. Each machine pool in a {hcp-title} cluster deploys in a single subnet within a single AZ. Each of these AZs can have only one machine pool. 
```